news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,75 @@
+
+<div align="center">
+  <video width="50%" controls>
+    <source src="https://assets.ppy.sh/media/news/coe_video_with_peppy.mp4?20230902" type="video/mp4" preload="none">
+  </video>
+</div>
+
+The biggest hint was peppy saying the solution may need to be altered because it depends on something that could be changed in the future. The only two possibilities were osu!(lazer) and the website. For those that guessed website and thought difficulty colors, they used the hint to it's fullest extent. There was a downside to this hint though. While peppy basically confirmed this theory about difficulty colors being correct, medal hunters also got a little mislead.
+
+After peppy said anyone, including new players would be able to get it, they started to believe it can't be performed on a difficulty higher than 4 star, which was completely wrong and resulted in community taking much more time to solve it. That subhint mentioning "Bask in the shade of an ailing mind", referring to basking in the **RGB** shades of an **Insane** diff in particular was completely ignored, because newcomers to osu! would not be able to get this medal.
+
+With a lot of guesses and such an insane 2 hints, 1 being slightly misleading, half of medal hunters entered full Sherlock Holmes mode with the possibilities of the solution, relying more on insanity than logic and drowning the discussion threads with abominations of theories. Some were genuinely good theories that Ephemeral would take note of, and others... well, let's just say Pac-Man theory.
+
+![](/wiki/shared/news/2023-08-28-medal-hunter-titles-second-coming/pacman_theory.jpg)
+
+But, despite all the challenges the community was still determined to find the solution. One of them was [UberFazz](https://osu.ppy.sh/users/8646059), who was going back and forth, seeking for the way to get this medal. At first, they assumed it would be achieving 16,777,216 (256 to the 3rd power) score, because that's how many possible colors there are in RGB. Another guess of theirs was to interact with the osu! website to find a hidden message somewhere on it, which would then lead to a solution. They were also told by one of developers they had insanely close guesses multiple times, but that was just not enough...
```

Comment:
```suggestion
But, despite all the challenges, the community was still determined to find the solution. One such participant, a fellow by the name of [UberFazz](https://osu.ppy.sh/users/8646059) had donned his fattest thinking cap, suggesting one theory of achieving 16,777,216 score, because that's how many possible colors there are in RGB (255 to the power of 3). Another guess of theirs was to interact with the osu! website to find a hidden message somewhere on it, which would then lead to a solution. They were also told by one of developers they had insanely close guesses multiple times, but it was all just not enough...
```